src/main/webapp/roles-assign.xhtml

```diff
@@ -31,7 +31,8 @@
                                         styleClass="DropdownPopup" panelStyleClass="DropdownPopupPanel"
                                         var="roleAssignee" itemLabel="#{roleAssignee.displayInfo.title}" itemValue="#{roleAssignee}" converter="roleAssigneeConverter">
                             <p:column>
-                                <h:outputText value="#{roleAssignee.displayInfo.title}"/> 
+                                <h:outputText value="#{roleAssignee.displayInfo.title}"/>
+                                <h:outputText value=" - #{roleAssignee.displayInfo.emailAddress}"/>
```

Comment:
@jeromeroucou I saw your note about this:

"Yes, there is a little difference from before : email addresses are now displayed next to the user's name, like this : John Doe - john.doe@example.com. I can remove that if needed."

I'm wondering... is this potentially a privacy problem? I can discover your email address by using the "add role" feature?

Reply:
Hi @pdurbin,

Indeed, a collection administrator could discover an email address with this feature. In that case, I'll remove my proposal.